src/org/pcic/DockerUtils.groovy

```diff
@@ -1,37 +1,54 @@
 package org.pcic
 
 import org.pcic.util.Utils
+import org.pcic.GitUtils
+import org.pcic.util.Global
+
 
 class DockerUtils implements Serializable {
     def steps
+    def utils
 
     DockerUtils(steps) {
         this.steps = steps
+        this.utils = new Utils(steps)
     }
 
-    void removeImage(String imageName) {
-        steps.sh(script: "docker rmi ${imageName}")
+    void removeImage(String image) {
+        steps.sh(script: "docker rmi ${image}")
     }
 
-    String buildImageName(String suffix) {
-        if (suffix.contains('/')) {
-            throw new Exception("Image name suffix: ${suffix} cannot contain `/` character")
+    String formatDockerString(String str, String replacement = '-') {
+        for (String illegalChar : Global.constants.illegalDockerChars) {
+            if (str.contains(illegalChar)) {
```

Comment:
Maybe we don't have to do this test? Just run the `replaceAll` and if there is nothing to replace it's a noop?

Reply:
Good idea!